test/Microsoft.Framework.PackageManager.Tests/PackageSourceUtilsFacts.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager.Tests
+{
+    public class PackageSourceUtilsFacts
+    {
+        [Theory]
+        [InlineData(@"C:\\foo\\bar", true)]
+        [InlineData(@".\foo\bar", true)]
+        [InlineData(@"foo\bar", true)]
+        [InlineData(@"/var/NuGet/packages", true)]
+        [InlineData(@"foo/bar", true)]
+        [InlineData(@"gopher://abc123", true)] // We won't actually use it, but we will think it's a path on the local FS :)
+        [InlineData(@"http://www.nuget.org", false)]
+        [InlineData(@"https://www.nuget.org", false)]
```

Comment:
You may want to add a case for paths starting in or containing ..\ (no idea what the name for this is), as it is related to issue i opened (https://github.com/aspnet/dnx/issues/2107) and unfortunately did not exemplify. An example would be ..\\..\LocalNugetRepo , which works well in previous versions.